lib/qesapdeployment.pm

```diff
@@ -2029,4 +2031,83 @@ sub qesap_az_create_sas_token {
     record_info('GENERATE-SAS', $cmd);
     return script_output($cmd);
 }
+
+=head2 qesap_terraform_clean_up_retry
+
+    qesap_terraform_clean_up_retry()
```

Comment:
Here it is a doc comment to show how the method has to be called. I know that this kind of doc comment is present for other sub in this file. But this particular sub does not have any argument so I think we do not need the document line 
 
```
qesap_terraform_clean_up_retry()
```

Reply:
I would leave it, as it shows the correct way to call the function is without arguments.